source/gui/settingsDialogs.py

```diff
@@ -1977,6 +1977,19 @@ def makeSettings(self, settingsSizer):
 		self.bindHelpEvent("KeyboardSettingsHandleKeys", self.handleInjectedKeysCheckBox)
 		self.handleInjectedKeysCheckBox.SetValue(config.conf["keyboard"]["handleInjectedKeys"])
 
+		minTimeout = int(config.conf.getConfigValidation(("keyboard", "multiPressTimeout")).kwargs["min"])
+		maxTimeout = int(config.conf.getConfigValidation(("keyboard", "multiPressTimeout")).kwargs["max"])
+		# Translators: The label for a control in keyboard settings to modify the timeout for a double keypress.
```

Comment:
This said double. Since it applies to more than just double pressed, I have suggested a change. The rest of the change was not for grammar, but just to try to keep it on one line. I'm not sure how exact this has to be for translation clarity, so you may want to ignore this suggestion.

```suggestion
		# Translators: Label of a control in keyboard settings to modify the timeout of a key pressed more than once.
```

Reply:
```suggestion
		# Translators: The label for a control in keyboard settings to modify the timeout for a multiple keypress.
```